config-scc.ini

```diff
@@ -0,0 +1,168 @@
+[global]
+datadir = ./datafiles
+timestamp-datadir = TRUE
+resultdir = ./results
+timestamp-resultdir = TRUE
+api = POSIX
+drop-caches = FALSE
+drop-caches-cmd = sudo -n bash -c "echo 3 > /proc/sys/vm/drop_caches"
+io-buffers-on-gpu = FALSE
+verbosity = 1
+scc = TRUE
+dataPacketType = timestamp
+
+[debug]
+stonewall-time = 30
+
+[ior-easy]
+API = 
+transferSize = 2m
+blockSize = 9920000m
+filePerProc = TRUE
+uniqueDir = FALSE
+run = TRUE
+verbosity = 
+
+[ior-easy-write]
+API = 
+run = TRUE
+
+[ior-rnd4K]
+API = 
+blockSize = 1073741824
+run = TRUE
```

Comment:
Is SCC running a standard set of tests or extended?  If standard, then all of the ior-rnd and mdworkbench sections could be removed, since they are not needed and are a bit confusing since they all have "run = true".